kclvm/sema/src/resolver/mod.rs

```diff
@@ -75,6 +75,7 @@ impl<'ctx> Resolver<'ctx> {
             }
             None => {}
         }
+        self.check_unused_import(pkgpath);
```

Comment:
move `check_unused_import` into `check_import` to call?

Reply:
has moved into lint check

Reply:
unused_import check has moved into lint check